### ISSUE TITLE
A11Y : Video Reveal Block on play functionality

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#901]
Since the Video Reveal content type expect users to 'opt-in' by actively pressing play to start the video, the expected behavior here should be to play the video with sound enabled.  I think we followed the video hero defaults here in playing the video muted by default but in this particular case it's actually better to play the video normally and with sound.
